test/built-ins/Temporal/PlainDate/prototype/monthCode/no-leap-months.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2025 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindate.prototype.monthcode
+description: iso8601 calendar does not have leap months
+features: [Temporal]
+---*/
+
+
+for (var year = 1970; year < 1975; year++) {
+  for (var month = 1; month < 13; month++) {
+    const date = Temporal.PlainDate.from({
+        year,
+        month, day: 1
+    });
```

Comment:
Let's keep the indentation consistent within each file, at least. (And maybe `day` can go on its own line)

Reply:
Same comment as above